jenkins.bash

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env bash
+set -e
+
+yarn install
+
+yarn build
```

Comment:
Where is this file used?

Reply:
Currently nowhere, I was anticipating this being used in Jenkins once we have a job set up though.

Reply:
It's easier to build using `docker build`.

FYI a few other projects we also added Makefile for convenience. For example: https://github.com/stellar/dashboard/blob/master/Makefile
It could be useful for development, for example you could add something like `make localrun` target to build and run the image locally.

Reply:
Good point. We'll probably use npm scripts to wrap it, make is unlikely to be more familiar for folks touching this code.